airflow/api_connexion/endpoints/task_endpoint.py

```diff
@@ -49,11 +51,16 @@ def get_task(dag_id, task_id):
         (permissions.ACTION_CAN_READ, permissions.RESOURCE_TASK_INSTANCE),
     ]
 )
-def get_tasks(dag_id):
+def get_tasks(dag_id, order_by='task_id'):
     """Get tasks for DAG"""
     dag: DAG = current_app.dag_bag.get_dag(dag_id)
     if not dag:
         raise NotFound("DAG not found")
     tasks = dag.tasks
+
+    try:
+        tasks = sorted(tasks, key=attrgetter(order_by.lstrip('-')), reverse=(order_by[0] == '-'))
```

Comment:
```suggestion
        tasks = sorted(tasks, key=attrgetter(order_by.lstrip('-')), reverse=(order_by[0:1] == '-'))
```
To avoid IndexError for an empty string.